modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/modal_content/AddFieldsModalContent.tsx

```diff
@@ -198,13 +198,18 @@ const AddFieldsModalContent = ({
 	const saveFDSFields = async () => {
 		setSaveButtonDisabled(true);
 
-		const creationData: Array<{name: string; type: string}> = [];
+		const creationData: Array<{
+			name: string;
+			sortable: boolean;
+			type: string;
+		}> = [];
 		const deletionIds: Array<number> = [];
 
 		visit(initialFields || [], (field: IFieldTreeItem) => {
 			if (selectedKeys.has(field.name) && !field.savedId) {
 				creationData.push({
 					name: field.name,
+					sortable: !!field.sortable,
```

Comment:
Do we need `!!`? Isn't `field.sortable` already a boolean?

Reply:
TypeScript complains as `sortable` is defined as optional (List and Cards are do not need it) and technically it can be `undefined`. 
I can change it to use a default value, like we are doing with [`type` ](https://github.com/liferay-frontend/liferay-portal/pull/4163/files/7da2ecec13ced266537f68b3bc8fd9846e037df0#diff-fcf5b4177264b13fbca505acf9d5f217027cf5471c57e6f26b9e864af2351a32R213).

Reply:
OK, this if fine then. I'd prefer the default value, like you mentioned in https://github.com/liferay-frontend/liferay-portal/pull/4163#discussion_r1584561687, simply because when I see `!!` I think string to boolean conversion. But that may be just me :man_shrugging: